orchestrator/session.go

```diff
@@ -256,12 +256,25 @@ func (s *Session) DestroyPhonon(keyIndex uint16) (privKey *ecdsa.PrivateKey, err
 }
 
 func (s *Session) IdentifyCard(nonce []byte) (cardPubKey *ecdsa.PublicKey, cardSig *util.ECDSASignature, err error) {
+	if !s.verified() {
+		return nil, nil, card.ErrPINNotEntered
+	}
```

Comment:
This if !s.verified() block should be removed unless there's some specific reason for it's inclusion. It's been left out of IdentifyCard intentionally since this may need to be called before a card is unlocked or even initialized, and it doesn't reveal or modify any sensitive information so it's safe to allow.